app/views/layouts/application.html.erb

```diff
@@ -180,7 +180,7 @@
   <% if @selected_branding_group.present? && @selected_branding_group.footer_image.file %>
     <%= image_tag @selected_branding_group.footer_image.url, class: 'pull-left' %>
   <% else %>
-    <%= link_to image_tag('broad-logo-white.png'), "https://www.broadinstitute.org", class: 'pull-left' %>
+    <%= link_to image_tag('broad-logo-white.png', alt:"Broad logo"), "https://www.broadinstitute.org", class: 'pull-left' %>
```

Comment:
```suggestion
    <%= link_to image_tag('broad-logo-white.png', alt:"Broad Institute logo"), "https://www.broadinstitute.org", class: 'pull-left' %>
```

Reply:
it could, but I would just as soon avoid multiple return values if we can help it.  it can update the properties in-place, which is why the method is called 'update...'    A more purely functional approach might, as Eric said, be a method called `getEventPropsWithTabsApplied()` that returns both and leaves the passed-in props unchanged.  Both are totally reasonable approaches

Reply:
The words in the image are "Broad Institute" so I'd like to leave it as is. Also I imagine that a screenreader would read out Broad as the word pronunciation aka /brôd/ so I think having the word "Institute" as well would help clarify this logo is for the institute rather than for something wide 😅